index.js

```diff
@@ -181,6 +181,7 @@ const ScrollableTabView = React.createClass({
       activeTab: this.state.currentPage,
       scrollValue: this.state.scrollValue,
       containerWidth: this.state.containerWidth,
+      defaultTabBarStyle: this.props.defaultTabBarStyle
```

Comment:
I don't want to pass props to ```tabBar``` throw ```ScrollableTabView```.
IMO good api would be
```
<ScrollableTabView renderTabBar={() => <DefaultTabBar someProp={'here'} />}>
...
</ScrollableTabView>
```

In future I'll remove props like ```tabBarUnderlineColor``` ```tabBarBackgroundColor``` from ```ScrollableTabView``` and they will be used in example.